dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -42,32 +44,29 @@ def main(project: str, ckpt_dir_path: str, save_only_latest: bool, ckpt_restore_
     max_epochs_save = os.environ.get("MAX_EPOCHS_SAVE", 5)
     max_steps_save = os.environ.get("MAX_STEPS_SAVE", 3)
 
-    trainer = Trainer(default_root_dir=ckpt_dir_path,
-                      plugins=[],
-                      callbacks=[checkpoint_callback],
-                      min_epochs=min_epochs_save,
-                      max_epochs=max_epochs_save,
-                      max_steps=max_steps_save,
-                      accelerator="gpu",
-                      strategy=dataflux_strategy,
-                      devices=1,
-                      num_nodes=15,
-                      )
+    trainer = Trainer(
+        default_root_dir=ckpt_dir_path,
+        plugins=[],
+        callbacks=[checkpoint_callback],
+        min_epochs=min_epochs_save,
+        max_epochs=max_epochs_save,
+        max_steps=max_steps_save,
+        accelerator="gpu",
+        strategy=dataflux_strategy,
+        devices=1,
+        num_nodes=15,
+    )
     trainer.fit(model, dataloader)
     start = time.time()
     for i in range(max_steps_save):
-        trainer.save_checkpoint(os.path.join(
-            ckpt_dir_path, f'checkpoints/ckpt_{i}.ckpt/'))
+        trainer.save_checkpoint(
+            os.path.join(ckpt_dir_path, f'checkpoints/ckpt_{i}.ckpt/'))
     end = time.time()
-    if torch.distributed.get_rank() == 0:
```

Comment:
It may still be worth printing something here (or even after each individual checkpoint is saved/loaded) so that it's clear to the user whether the benchmark is making progress when they run it

Reply:
Agreed! I will put print statements in appropriate places to log progress. I've made a couple more minor but necessary changes to this file which I want to include in a different PR. I will add the print statements too to the new set of commits.